src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -7,6 +7,95 @@
 
 namespace System.Text.Json
 {
+    public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.Generic.ICollection<System.Text.Json.JsonNode>, System.Collections.Generic.IEnumerable<System.Text.Json.JsonNode>, System.Collections.Generic.IList<System.Text.Json.JsonNode>, System.Collections.IEnumerable
```

Comment:
Do we need to explicitly specify all of these interfaces? Isn't IList<T> already implementing a bunch of these?

Reply:
It's how `ref` file was generated. In `JsonArray.cs` there are only `JsonNode` and `IList<JsonNode>` .

Reply:
An opportunity for improving GenAPI: https://github.com/dotnet/arcade/blob/e1bbfbb27da0a3c00be6e8a874731b4211feb052/src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Types.cs#L94  maybe we should file an issue.